pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1479,6 +1506,36 @@ func SetDefaults(cr *api.KieApp) {
 	cr.Status.Applied = *specApply
 }
 
+func checkTerminationRouteOnProcessMigration(migration *api.ProcessMigrationObject) {
+	if migration.TerminationRoute == nil {
+		migration.TerminationRoute = &api.TerminationRoute{}
+	}
+}
+
+func checkTerminationRouteOnDashbuilder(dashbuilder *api.DashbuilderObject) {
+	if dashbuilder.TerminationRoute == nil {
+		dashbuilder.TerminationRoute = &api.TerminationRoute{}
+	}
+}
+
+func checkTerminationRouteOnServer(server *api.KieServerSet) {
+	if server.TerminationRoute == nil {
+		server.TerminationRoute = &api.TerminationRoute{}
+	}
+}
+
+func checkTerminationRouteOnConsole(console *api.ConsoleObject) {
+	if console.TerminationRoute == nil {
+		console.TerminationRoute = &api.TerminationRoute{}
+	}
+}
+
+func checkTerminationRouteOnSmartRouter(smartrouter *api.SmartRouterObject) {
+	if smartrouter.TerminationRoute == nil {
+		smartrouter.TerminationRoute = &api.TerminationRoute{}
+	}
+}
```

Comment:
For what exactly are these checks needed? Are they configuring termination (adding termination property into kieapp) by default - if so this is probably something that we won't do in operator.

Reply:
ok